webapp/locales/de.json

```diff
@@ -498,9 +498,9 @@
   },
   "profile": {
     "commented": "Kommentiert",
-    "follow": "Folgen",
-    "followers": "Folgen",
-    "following": "Folge Ich",
+    "follow": "Abonnieren",
+    "followers": "Abonnieren",
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-21T11:23:32Z" title="Friday, February 21st 2020, 12:23:32 pm +01:00">Feb 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -498,9 +498,9 @@
   },
   "profile": {
     "commented": "Kommentiert",
-    "follow": "Folgen",
-    "followers": "Folgen",
-    "following": "Folge Ich",
+    "follow": "Abonnieren",
+    "followers": "Abonnieren",
```


Once again, this sounds strange. `Wird abonniert von` ?